### PR TITLE
fix: polaris, nova, pluto CVE fix

### PR DESCRIPTION
**Pull request by SandhyaRavi2403**
**What problem does this PR solve?**:
Patch Polaris, Nova and Pluto image with fixed CVE's 

workflow link: 
https://github.com/mesosphere/dkp-container-images/actions/runs/9906109203
https://github.com/mesosphere/dkp-container-images/actions/runs/9906055964
https://github.com/mesosphere/dkp-container-images/actions/runs/9906012179

**Which issue(s) does this PR fix?**:

https://jira.nutanix.com/browse/NCN-101394

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

**Comment by SandhyaRavi2403** [comment]
@mhrabovcin, I am sure if the ref is correct. getting the below error
`2024-07-12 14:55:40 ERR https://github.com/FairwindsOps/pluto@v5.10.6-d2iq.0 (failed to access git repository: unable to get URL "https://api.github.com/repos/FairwindsOps/pluto/contents/?ref=v5.10.6-d2iq.0": 404 Not Found )    err=<nil>
2024-07-12 14:55:40 ERR https://github.com/FairwindsOps/polaris@7.0.2-d2iq.0 (failed to access git repository: unable to get URL "https://api.github.com/repos/FairwindsOps/polaris/contents/?ref=7.0.2-d2iq.0": 404 Not Found )    err=<nil>
2024-07-12 14:55:40 ERR https://github.com/FairwindsOps/nova@3.4.0-d2iq.0 (failed to access git repository: unable to get URL "https://api.github.com/repos/FairwindsOps/nova/contents/?ref=3.4.0-d2iq.0": 404 Not Found )    err=<nil>`

File: licenses.d2iq.yaml [hunk 1]
```diff
@@ -470,20 +470,20 @@ resources:
       - license_path: COPYING
         ref: ${image_tag}
         url: https://github.com/ceph/ceph
-  - container_image: quay.io/fairwinds/nova:3.4.0
+  - container_image: ghcr.io/mesosphere/dkp-container-images/quay.io/fairwinds/nova:3.4.0-d2iq.0
     sources:
       - license_path: LICENSE
-        ref: ${image_tag}
```
Comment (mhrabovcin) [inline licenses.d2iq.yaml:476]: For the tag matching we are using the https://github.com/drone/envsubst which supports functions - https://github.com/drone/envsubst?tab=readme-ov-file#supported-functions

We can use `${var%pattern}` to remove suffix of the tag and keep it dynamic. In this particular case it would be:

```
ref: ${image_tag%-d2iq.0}
```

Comment (SandhyaRavi2403) [reply licenses.d2iq.yaml:476]: updated!

File: services/git-operator/0.1.0/git-operator-manifests/all.yaml [hunk 2]
```diff
@@ -699,7 +699,7 @@ spec:
         - --metrics-bind-address=127.0.0.1:8080
         - --leader-elect
         - --namespace=${NAMESPACE:=git-operator-system}
-        image: docker.io/mesosphere/git-operator:v0.8.2@sha256:7dde5a0836d212d8fe31f65c42bec32c91962858f6cb7da5526a0b60727141fb
+        image: docker.io/mesosphere/git-operator:v0.8.3@sha256:3a4c42890549394bbe3842d99c7bf143334c19dc6e8d13413cd796c219aead8e
```
Comment (mhrabovcin) [inline services/git-operator/0.1.0/git-operator-manifests/all.yaml:702]: Can you rebase your PR, this change seems irrelevant.

Comment (SandhyaRavi2403) [reply services/git-operator/0.1.0/git-operator-manifests/all.yaml]: Updated !!

Comment (SandhyaRavi2403) [reply services/git-operator/0.1.0/git-operator-manifests/all.yaml]: @mhrabovcin Review comments addressed

**Review by mhrabovcin — APPROVED** [review]

**Comment by SandhyaRavi2403** [comment]
@mhrabovcin , The corresponding Kommander PR `https://github.com/mesosphere/kommander/pull/4815` has all tests successful except the airgapped validation. Do we still need to wait for airgapped as well to pass ?

File: services/dkp-insights/1.1.2-d2iq.0/defaults/cm.yaml
Comment (mhrabovcin) [inline services/dkp-insights/1.1.2-d2iq.0/defaults/cm.yaml:1]: I think we need to rename this configmap.

Comment (SandhyaRavi2403) [reply services/dkp-insights/1.1.2-d2iq.0/defaults/cm.yaml:1]: Should it be renamed to 1.1.3 ?

Comment (mhrabovcin) [reply services/dkp-insights/1.1.2-d2iq.0/defaults/cm.yaml:1]: This is not limited to this PR but in general, whenever values in the default CM are changed, like in this case, the CM name needs to be updated.

https://github.com/mesosphere/kommander-applications/blob/main/services/dkp-insights/1.1.2/defaults/cm.yaml#L4

```
  name: dkp-insights-1.1.2-d2iq-defaults
```

to

```
  name: dkp-insights-1.1.2-d2iq.0-d2iq-defaults
```

The naming scheme is `<service>-<version>-d2iq-defaults`

This new name needs to be reflected in the `HelmRelease` that references this CM.

https://github.com/mesosphere/kommander-applications/blob/main/services/dkp-insights/1.1.2/helmrelease/helmrelease.yaml#L27

Comment (SandhyaRavi2403) [reply services/dkp-insights/1.1.2-d2iq.0/defaults/cm.yaml:1]: Updated
